### PR TITLE
ZD-4769105: Change alignment of L&G NHS Logo

### PR DESCRIPTION
**Pull request by dj-maisy**
## What?
This is an amendment to the Lewisham & Greenwich NHS Trust custom branding CSS to swap around the Logo and Service Title elements - the Logo asset in question is designed to be right-aligned and the Sass changes included should rectify this.

The `float:right;` should be applied to all display sizes as this will also put it in the right place on mobile.

## Related Items
https://github.com/alphagov/pay-js-commons/pull/776

### CODE REVIEW

**Comment by dj-maisy** [comment]
This change is a follow-on from #776

File: sass/custom/lewisham-and-greenwich-nhs-trust.scss [hunk 1]
```diff
@@ -8,12 +8,12 @@ $govuk-font-family: -apple-system,BlinkMacSystemFont,helvetica neue,helvetica,ub
 $custom-banner-colour: #005EB8;
 $custom-banner-border-colour: #ffffff;
 $custom-text-colour: #ffffff;
-$logo-image-height: 2em;
+$logo-image-width: 215px;
```
Comment (oswaldquek) [inline sass/custom/lewisham-and-greenwich-nhs-trust.scss:11]: Maybe I'm not the best person to review this, but why did you change `logo-image-height` to `logo-image-width`? See this [commit](https://github.com/alphagov/pay-js-commons/commit/1700ebb936e38b7a09348c563e351dec2019d6e0) that right aligns the logo too.

Comment (dj-maisy) [reply sass/custom/lewisham-and-greenwich-nhs-trust.scss:11]: Our internal documentation advises us to do this.

**Review by iqbalgds — APPROVED** [review]
